frontend/src/components/ChooseLevelPage/ChooseLevelPage.scss

```diff
@@ -1,43 +1,49 @@
 @import "../../variables.scss";
-ul {
-  margin: 0;
-  padding: 0;
+.cardContainer {
   display: flex;
-  flex-direction: row;
-  width: 100vh;
-}
-li {
-  color: $primaryColor;
-  list-style: none;
-  border: 2px solid $secondaryColor;
-  text-align: center;
-  position: relative;
+  width: 100vw;
+  height: 100vh;
+  &__playButton {
+    margin: 3rem;
+    background-color: $primaryColor;
+    font-size: 1.3rem;
+    font-weight: bold;
+    width: 10vw;
+    height: 6vh;
+    border-radius: 10% 30% 50% 70%;
+  }
 }
 .card {
-  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  color: $primaryColor;
+  border-inline: 1px solid $primaryColor;
   background-size: cover;
   width: 20vw;
-}
-button {
-  background-color: $primaryColor;
-  font-size: 1.3rem;
-  font-weight: bold;
-  width: 10vw;
-  height: 3vw;
-  border-radius: 10% 30% 50% 70%;
-  a {
-    text-decoration: none;
-    color: $secondaryColor;
+  &CRÉTACÉ {
```

Comment:
No Franglais

Reply:
We named our className according to the BEM method, which induces the written words based on the display of the table name " universe = [    { name: "CRÉTACÉ" },    { name: "ROMAIN" },    { name: "EGYPTE" },    { name: "MOYEN-AGE" },    { name: "MODERNE" }  ];     Is this valid?